account-validator-api/src/java/org/sakaiproject/accountvalidator/logic/ValidationLogic.java

```diff
@@ -100,8 +110,8 @@
 	 * @param toSave
 	 */
 	public void save(ValidationAccount toSave);
-	
```

Comment:
Can you not make un-needed whitspace changes please as it means things don't merge as cleanly as they could.